submissions/7SergSmith7/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,791 @@
+            <div class="selected-seats-info__seats-list__row">
+              <div class="seats-list__row__seats">
+                <div class="row__seats place">
+                  <span class="place__number">3</span>
+                  <span class="place__title">Place</span>
+                </div>
+                <div class="row__seats row-seats">
+                  <span class="row-seats__number">8</span>
+                  <span class="row-seats__title">Row</span>
+                </div>
+                <div class="row__seats seat-cost">
+                  <span class="seat-cost__number">$5</span>
+                  <span class="seat-cost__title">Cost</span>
+                </div>
+              </div>
+              <span class="row__delete-icon">X</span>
+            </div>
+            <div class="selected-seats-info__seats-list__row">
+              <div class="seats-list__row__seats">
+                <div class="row__seats place">
+                  <span class="place__number">3</span>
+                  <span class="place__title">Place</span>
+                </div>
+                <div class="row__seats row-seats">
+                  <span class="row-seats__number">8</span>
+                  <span class="row-seats__title">Row</span>
+                </div>
+                <div class="row__seats seat-cost">
+                  <span class="seat-cost__number">$5</span>
+                  <span class="seat-cost__title">Cost</span>
+                </div>
+              </div>
+              <span class="row__delete-icon">X</span>
+            </div>
+          </div>
+          <div class="selected-seats-info__total-cost-block">
+            <span class="selected-seats-info__total-cost-block__text"
+              >Final cost</span
+            >
+            <span class="selected-seats-info__total-cost-block__total-cost"
+              >$10</span
+            >
+          </div>
+          <a href="#" class="selected-seats-info__checkout-btn">checkout</a>
+        </div>
+      </section>
+      <section class="main__screen-and-seats-block">
+        <nav class="main__screen-and-seats-block__navigation">
+          <div class="main__screen-and-seats-block__navigation__date">
+            <ul class="navigation__date__list">
+              <li class="date__list__item navigation-arrow"><</li>
+              <li class="date__list__item selected-date">02 February</li>
+              <li class="date__list__item navigation-arrow">></li>
+            </ul>
+          </div>
+          <div class="main__screen-and-seats-block__navigation__time">
+            <ol class="navigation__time__list">
+              <li class="time__list__item">10:00</li>
+              <li class="time__list__item">11:20</li>
+              <li class="time__list__item selected-time">12:00</li>
+              <li class="time__list__item">16:00</li>
+              <li class="time__list__item">17:20</li>
+              <li class="time__list__item">19:00</li>
```

Comment:
Logically these elements are intended for choosing a movie session time, which means you need to choose interactive elements for this.

Reply:
Do I need to use a button or radio?

Reply:
done